google-summer-of-code/gsoc-2022.md

```diff
@@ -80,8 +83,9 @@ Contributors are welcome to participate in any talks and discussions in the Knat
 
 * Knative users group: https://groups.google.com/g/knative-users
     * Preferred communication channel. Introduce yourself here as a candidate contributor.
-* Slack: https://slack.knative.dev/ Explore the channels yourselves, however, make sure you're checking these:
-    * `#serving`: Knative Serving related discussions
-    * `#eventing`: Knative Eventing related discussions
+* Slack: https://slack.cncf.io/ Explore the channels yourselves, however, make sure you're checking these:
+    * [`#knative-serving`](https://cloud-native.slack.com/archives/C04LMU0AX60): Knative Serving related discussions
+    * [`#knative-eventing`](https://cloud-native.slack.com/archives/C04LMU33V1S): Knative Eventing related discussions
+    * [`#knative-functions`](https://cloud-native.slack.com/archives/C04LKEZUXEE): Knative Functions related discussions
```

Comment:
Shoot, I'm sorry, I completely missed that this was last year's GSOC doc... @aliok should we update the slack links for this one or just leave as is (since it's a historical doc at this point)?

Reply:
@psschwei I would just leave as is, but it is ok; there's no need to revert.

Reply:
no big issue, to point to new place